src/modules/admin/components/formRules/FormRule.tsx

```diff
@@ -143,4 +122,57 @@ const FormRule: React.FC<Props> = ({ rule, formRole, actionButtons }) => {
   );
 };
 
-export default FormRule;
+interface FormRuleProps {
+  rule: FormRuleFieldsFragment;
+  formRole?: FormRole;
+  actionButtons?: ReactNode;
+}
+export const FormRule: React.FC<FormRuleProps> = ({
+  rule,
+  formRole,
+  actionButtons,
+}) => {
+  return (
+    <BaseFormRule
+      project={rule.project?.projectName}
+      projectType={
```

Comment:
I'm seeing "project type is undefined" after selecting a project type:

<img width="872" alt="Screenshot 2024-08-09 at 10 33 25 AM" src="https://github.com/user-attachments/assets/15edd828-0ed5-4561-b189-7a53ab34391a">

Reply:
@gigxz  I wasn't able to reproduce this locally. Are you still seeing it?